en/docs/references/concepts/authorization/jarm.md

```diff
@@ -0,0 +1,177 @@
+# JWT Secured Authorization Response Mode (JARM) for OAuth 2.0
+
+The [JWT Secured Authorization Response Mode for OAuth 2.0 (JARM) specification](https://openid.net/specs/openid-financial-api-jarm-ID1.html) defines new JWT-based modes to encode OAuth2 authorization responses. These modes allow clients to request authorization response parameters and additional data in JWT format.
+
+## Authorization flow
+
+Below is a sample authorization request sent to the authorization endpoint of the WSO2 Identity Server.
+
+```bash
+https://<IS_HOME>/oauth2/authorize?prompt=login&scope=openid&redirect_uri=https://<CLIENT_HOST>/redirects/redirect1&client_id=<CLIENT_ID>&response_type=<RESPONSE_TYPE>&response_mode=<RESPONSE_MODE>
+```
+
+The parameters used in the authorization request are defined below.
+
+<table>
+    <tr>
+        <th>Parameter</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <th>
+            <code>reponse_type</code>
+        </th>
+        <td>
+            Informs the WSO2 Identity Server of the desired OAuth2 authorization processing flow, which determines the type of authorization information that should be returned.</br></br>
+            Possible values:
+            <ul>
+                <li>
+                    <code>code</code>: Used if the <a href="../authorization-code-grant">authorization code flow</a> is implemented.
+                </li>
+                <li>
+                    <code>token</code>: Used if the <a href="../implicit-grant">implicit flow</a> is implemented.
+                </li>
+                <li>
+                    <code>code id_token</code>: Used if the <a href="../openid-connect-hybrid-flow">hybrid flow</a> is implemented.
+                </li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <th>
+            <code>response_mode</code>
+        </th>
+        <td>
+            Specifies how the authorization information should be returned to the client. By default, the following response modes are supported in WSO2 Identity Server:
+            <ul>
+                <li><code>query</code></li>
+                <li><code>fragment</code></li>
+                <li><code>form_post</code></li>
+            </ul>
+            In the default authorization flow (for default response modes), the authorization response parameters, such as the authorization code and ID token, are sent as human-readable plain text to the redirect URL, as shown below.</br></br>
+            <code>https://&lt;CLIENT_HOST&gt;/redirects/redirect1?code=&lt;AUTH_CODE&gt;&session_state=&lt;SESSION_STATE&gt;</code></br></br>
+            The JARM specification is used to send this response more securely by encoding the response parameters in a JWT.
+        </td>
+    </tr>
+</table>
+
+## Enable JARM
```

Comment:
Took this to the top before `Using JARM`. hope it's okay?

Reply:
yep. that's actually better